Library/Homebrew/missing_formula.rb

```diff
@@ -101,7 +101,7 @@ def tap_migration_reason(name)
           EOS
           break if new_tap_name == CoreTap.instance.name
 
-          install_cmd = if new_tap_user == "caskroom"
+          install_cmd = if new_tap_user == "caskroom" || (new_tap_user == "homebrew" && new_tap_repo.start_with?("cask"))
```

Comment:
Feels a bit much for one line

Reply:
For what it's worth, this is only temporary.

Reply:
How temporary, roughly?

Reply:
It can be removed after the migration.

Reply:
Exactly.

Reply:
I concur.